go.mod

```diff
@@ -302,7 +302,7 @@ replace (
 
 replace (
 	github.com/containerd/containerd => github.com/containerd/containerd v1.5.18
-	github.com/docker/distribution => github.com/docker/distribution v2.8.1+incompatible
+	github.com/docker/distribution => github.com/docker/distribution v2.8.2+incompatible
```

Comment:
i wonder if this replace can be removed here 
but this is just nice to have, not mandatory